product_harmonized_system/models/product_template.py

```diff
@@ -20,16 +20,17 @@ class ProductTemplate(models.Model):
         "http://www.wcoomd.org/. You can leave this field empty "
         "and configure the H.S. code on the product category.",
     )
+
+
+class ProductProduct(models.Model):
+    _inherit = "product.product"
+
```

Comment:
I am not against moving the origin_country_id to the variant but in that I think we consider really moving this field, not an extra feild on top of the product.template since this could be hard to manage.
Why not turn the origin_country_id  on product.template into a computed field like odoo does with the default_code field, cf. https://github.com/odoo/odoo/blob/1b57cf252dc868c3b4ff59894fad39f2e955dab5/addons/product/models/product_template.py#L146